### ISSUE TITLE
PHP opening tag inside javaScript script tags results in parse error.

### ISSUE THREAD

**Issue by wowcut** [BenoitZugmeyer/eslint-plugin-html#75]
I have some code that looks like this: 
```
<body>
  <script>
<?php 
...
```

which will produce parse errors with this plugin: 
```Parsing error: Unexpected token ? ```

Yes, I know that is very ugly code and a bad practice, this is some abandoned legacy code and that is the reason why I really would love to use this plugin to check all the files with that ugly code without touching manually. Would be ignoring everything `<?php ?>` tags a good thing?

**Comment by BenoitZugmeyer** [comment]
It is not necessary a bad practice, don't worry! Just make sure to sanitize the generated script. Since it is a popular issue, I added a [*Troubleshooting* section](https://github.com/BenoitZugmeyer/eslint-plugin-html#linting-templates-or-php) in the README. Sadly, you will have to edit the files manually. I can't blindly remove PHP tags, because:
* if I remove PHP tags, I would be asked to remove tags for other template engines, and writing parsers for everything would be too much work
* even if the tags are removed, the resulting code is unlikely to be valid JS anyway. Example for the most common pattern:
```html
<script>
var data = <?php json_encode($data); ?>;
</script>
```
would be rewritten to
```html
<script>
var data = ;
</script>
```
